routingpy/routers/google.py

```diff
@@ -291,6 +331,18 @@ def _parse_direction_json(response, alternatives):
             else:
                 return Direction()
 
+        status = response["status"]
+
+        if status in STATUS_CODES.keys():
+            error = (
+                RouterServerError
+                if status == "UNKNOWN_ERROR"
+                else OverQueryLimit
+                if status in ["OVER_QUERY_LIMIT", "OVER_DAILY_LIMIT"]
+                else RouterApiError
```

Comment:
sorry, for more than 2 choices it's really better to do if elses as you did before. this is fairly hard to read, esp with this (black) formatted stuff..